src/components/AppNavi/AppNaviButton.tsx

```diff
@@ -74,8 +74,19 @@ const CurrentWrapper = styled.span<{ themes: Theme }>`
 
     return css`
       ${BaseStyle}
-      border-bottom: ${size.pxToRem(3)} solid ${palette.MAIN};
       color: ${palette.TEXT_BLACK};
+      position: relative;
+
+      &:after{
```

Comment:
これ lint で弾いてよさそう。疑似要素と疑似クラスは分けたい。

```suggestion
      &::after{
```